### PR TITLE
Fix/coverage

### PR DESCRIPTION
**Pull request by frcroth**

### CODE REVIEW

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st [hunk 1]
```diff
@@ -0,0 +1,10 @@
+unit-tests
+testInspectDocument
+	
+	| possibleMorphs |
+	self editorModel inspectDocument.
+	possibleMorphs := Project current world submorphs.
+	possibleMorphs := possibleMorphs select: [:morph | morph class = PluggableSystemWindow].
+	possibleMorphs := possibleMorphs select: [:morph | morph label = self document class name].
+	possibleMorphs := possibleMorphs select: [:morph | morph model object = self editorModel targetDocument].
+	self assert: (possibleMorphs notEmpty)
```
Comment (j-hellenberg) [inline packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st:10]: Explain pls

File: packages/RichTextEditing-Tests.package/RichTextEditorTextTests.class/class/lastStoredRun.st [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
... add: #testUS36StructureIsNotExtendable; add: #testUS49ApplySelectedStructureWithoutASelectionToNextTextBeginning; add: #testUS49NoStructureApplyWhenCaretMovedEnd; add: #testUS23TriangleReappearsWhenEditing; add: #testUS49ApplySelectedStructureWithoutASelectionToNextTextEnd; add: #testUS49NoStructureApplyWhenCaretMovedMid; yourself)); add: (#failures->((Set new))); add: (#timeStamp->'12 June 2020 7:50:05.529548 pm' asTimeStamp); add: (#durations->((Dictionary new) add: (#testUS23TriangleReappearsWhenEditing->1582); add: (#testUS49ApplySelectedStructureWithoutASelectionToNextTextBeginning->795); add: (#testUS49NoStructureApplyWhenCaretMovedEnd->795); add: (#testUS49NoStructureApplyWhenCaretMovedBeginning->787); add: (#testUS23WhenTriangleIsVisibleCtrlSCanBePressedForSaveDialog->780); add: (#testUS49ApplySelectedStructureWithoutASelectionToNextTextMiddle->783); add: (#testUS36StructureIsNotExtendable->781); add: (#testUS49NoStructureApplyWhenCaretMovedMid->795); add: (#testUS49ApplySelectedStructureWithoutASelectionToNextTextEnd->789); add: (#testUS23TriangleDisappearsAfterSuccesfulSave->932); add: (#testUS23TriangleIsStillVisibleWhenSaveDialogIsCancelled->899); yourself)); yourself)
```
Comment (j-hellenberg) [inline packages/RichTextEditing-Tests.package/RichTextEditorTextTests.class/class/lastStoredRun.st:3]: Where did all of those come from?

Comment (frcroth) [reply packages/RichTextEditing-Tests.package/RichTextEditorTextTests.class/class/lastStoredRun.st:3]: I don't know should be removed

File: packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st (hunk 1 above)
Comment (frcroth) [reply packages/RichTextEditing-Tests.package/RichTextEditorTests.class/instance/testInspectDocument.st:10]: Check if the Document Inspector is opened. 
Inspect all open Morphs. Select for pluggableSystemWindow, of which Inspector is an object. Other morphs may not understand the message label. Then select the window which is the correct inspect Document

**Pull request merged**